app/js/populateTestDropdown.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
does this need its own file? why not put this into the code for adding a question and save creating a new JS file

Reply:
Have made this more of a general purpose function, this is because Story4 build accordion task will do something very similar to this

The general purpose function sits in its own file, the function is called in the sendQuestionFormOutput file